src/toxic.c

```diff
@@ -837,6 +888,7 @@ static Tox *load_tox(char *data_path, struct Tox_Options *tox_opts, Tox_Err_New
 
         if (store_data(m, data_path) == -1) {
             exit_toxic_err("failed in load_tox", FATALERR_FILEOP);
+            return NULL;
```

Comment:
You added a lot of `return NULL;` after `exit_toxic_err` calls. Don't they exit?

Reply:
I added those to see if they would get rid of a strange codefactor warning, which they did. So I can either leave them (harmless) or @iphydf can fix the false positive.

Reply:
Going to leave them in even though CodeFactor no longer is a required check?